<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>452</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Фигуры</string>
    </property>
    <addaction name="actionSquare"/>
    <addaction name="actionCircle"/>
    <addaction name="separator"/>
    <addaction name="actionChooseColor"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Опции</string>
    </property>
    <addaction name="actionCustomize"/>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCircle"/>
   <addaction name="separator"/>
   <addaction name="actionSquare"/>
   <addaction name="separator"/>
   <addaction name="actionTriangle"/>
   <addaction name="separator"/>
   <addaction name="actionSomeTriangle"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSquare">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/girpodn_.jpg</normaloff>
     <normalon>:/girpodn_.jpg</normalon>:/girpodn_.jpg</iconset>
   </property>
   <property name="text">
    <string>Квадрат</string>
   </property>
   <property name="toolTip">
    <string>Добавить квадрат</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Круг</string>
   </property>
   <property name="toolTip">
    <string>Добавить круг</string>
   </property>
  </action>
  <action name="actionChooseColor">
   <property name="text">
    <string>Выбрать цвет</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Удалить фигуру</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Загрузить</string>
   </property>
  </action>
  <action name="actionCustomize">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionTriangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Тр</string>
   </property>
   <property name="toolTip">
    <string>Выберите треугольник</string>
   </property>
  </action>
  <action name="actionSomeTriangle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Л.Тр.</string>
   </property>
   <property name="toolTip">
    <string>Ввод любого треугольника.</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
